iam_apikey.go

```diff
@@ -47,10 +47,25 @@ func (ListAPIKeys) Response() interface{} {
 	return new(ListAPIKeysResponse)
 }
 
+// ListAPIKeyOperations represents a search for operations for the current API key
```

Comment:
Is it really necessary to implement this call? Doesn't the `getAPIKey` also return the key's supported operations?

Reply:
There is a difference.

ListApiKeyOperations returns all of the things a key can do (even for a key without restrictions - you get the full list), whereas GetApiKey returns the details of the policy for the key - so for an unrestricted key no operations would be returned.

ListApiKeyOperations is primarily for the UI so we can display a list of checkboxes when creating a new key, it's debatable how useful this is from the CLI as the operations do not always align with the commands exposed by the CLI. If we were to do some autocomplete on the operations from the CLI you could use it.